jetstream/allocation.py

```diff
@@ -285,6 +286,9 @@ def get_or_create_allocation_source(api_allocation, update_source=False):
 def find_user_allocation_source_for(driver, user):
     tacc_user = driver.get_tacc_username(user)
     # allocations = driver.find_allocations_for(tacc_user)
+    if not tacc_user:
+        return []
+
```

Comment:
Okay, so we'll return an empty list when we've encountered an XSEDE to TACC lookup. 

Looks good.